src/video/cocoa/SDL_cocoawindow.m

```diff
@@ -1738,13 +1738,19 @@ - (BOOL)acceptsFirstMouse:(NSEvent *)theEvent
         return SDL_SetError("%s", [[e reason] UTF8String]);
     }
 
+    [nswindow setColorSpace:[NSColorSpace sRGBColorSpace]];
+
 #if MAC_OS_X_VERSION_MAX_ALLOWED >= 101200 /* Added in the 10.12.0 SDK. */
     /* By default, don't allow users to make our window tabbed in 10.12 or later */
     if ([nswindow respondsToSelector:@selector(setTabbingMode:)]) {
         [nswindow setTabbingMode:NSWindowTabbingModeDisallowed];
     }
 #endif
 
+    if ([nswindow respondsToSelector:@selector(setReleasedWhenClosed:)]) {
+        [nswindow setReleasedWhenClosed:NO];
```

Comment:
This is already done in SDL's code: https://github.com/libsdl-org/SDL/blob/49d6f3e6c3aaa1596dd2cc770baa3dd85d4ad2a4/src/video/cocoa/SDL_cocoawindow.m#L1685

Reply:
Aha, I've rebased so many times I missed that, so thanks for catching that. I'll drop that patch.